static/src/javascripts/projects/common/modules/experiments/ab-tests.js

```diff
@@ -8,7 +8,6 @@ import { articlesViewedBanner } from 'common/modules/experiments/tests/contribs-
 import { xaxisAdapterTest } from 'common/modules/experiments/tests/commercial-xaxis-adapter';
 import { appnexusUSAdapter } from 'common/modules/experiments/tests/commercial-appnexus-us-adapter';
 import { pangaeaAdapterTest } from 'common/modules/experiments/tests/commercial-pangaea-adapter';
-import { signInGateSecundus } from 'common/modules/experiments/tests/sign-in-gate-first-test';
```

Comment:
Do you need to remove the file too or are you keeping it around?

Reply:
we're keeping it around, as we'll be likely running a similar tests in the near future